README.md

```diff
@@ -81,14 +80,51 @@ Fork and clone this repository. Issue pull-requests one document at a time.
 
 
  Finally, you can tell sculpin to watch the docs directory and automatically
- regenerate anything changed:
+ regenerate upon file modification:
  ```
  sculpin generate --server --watch
  ```
- If you use --watch and see it constantly running, regenerating, drop --watch
- until you identify and resolve the problem.  
-5. Images  
- For images to render on your local environment you need to apply these commands from within the `documentation` directory:
+ If changes cause Sculpin to continuously regenerate files, drop `--watch` until you investigate and debug.
+5. Create the following symlink for images to render on your local environment:
+
+ ```
+ $ cd output_dev
+ $ ln -s ./ source
+ ```
+
+## Local Setup: Git for Windows (Optional)
+
+1. If you do not already have PHP installed, use [Windows Web Platform Installer](https://www.microsoft.com/web/gallery/install.aspx?appid=PHP56) or [install manually](http://php.net/manual/en/install.windows.manual.php).
+2. Install [Git for Windows](https://git-for-windows.github.io/) using the default settings and open Git BASH.
+3. Clone the documentation repository.
+4. Install [Sculpin](https://sculpin.io) within the current users root directory (e.g. `/c/Users/username`):
+
+ ```
+ curl -O https://download.sculpin.io/sculpin.phar
+ chmod +x sculpin.phar
+ ```
+ Open the Start Menu and search for System (Control Panel), click to open. Select **Advanced System Setttings** then click **Environment Variables**. Select the **PATH** variable then click **Edit** and **New**. Enter the path for the current users root directory (e.g. `C:\Users\username`) and click **Okay**. Restart your machine.
```

Comment:
Also, they'll need to associate .phar files with the php.exe program. Windows will prompt them to do this the first time sculpin is run.